pkg/monitoring/vms/prometheus/prometheus.go

```diff
@@ -47,60 +50,85 @@ var (
 	storageIopsDesc = prometheus.NewDesc(
 		"kubevirt_vm_storage_iops_total",
 		"I/O operation performed.",
-		[]string{"domain", "drive", "type"},
+		[]string{
+			"node", "namespace", "name",
+			"domain", "drive", "type",
```

Comment:
I wonder if it makes sense to expose `domain` at all. It only exists because libvirt needs a name.

Reply:
My only concern is if removing it somehow breaks the backward compatibility. Otherwise, no problem at all. Should anyone ever need it, there are ways to get it.

Reply:
Hm, good point. Is there already e.g. UI integration?

Reply:
> Hm, good point. Is there already e.g. UI integration?

Let's ask @jelkosz about this!